apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -43,6 +45,10 @@ const getThemedStyles = themedStyleSheet((t: Theme) => {
   };
 });
 
+const EmptyComponent: React.FunctionComponent = () => {
+  return <RNText style={fluentTesterStyles.noTest}>Select a component from the left.</RNText>;
```

Comment:
Does this need to say something other than "left" in mobile?

Reply:
I think this is never rendered on mobile. You just see the test page. Good catch though.

Idea I've had: Replace empty test page with a short description of how to use tester. I.E:
- Look at platform status to see what's available where
- Play around with Theme, etc.

I'm happy to change the wording here to "Select a component from the test list" for now though, even if it doesn't affect mobile.

Reply:
Fixed from "left" -> "test list".